lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/DirectoryImplementor.java

```diff
@@ -112,6 +114,9 @@ long fileLength(final String name) {
     }
 
     IndexOutput createOutput(final String name) {
+       if(log.isDebugEnabled()) {
```

Comment:
This entire module needs re-formatting, let's not bring it in this PR to reduce noise